tests/test_yaml/version_jinja2sha.yaml

```diff
@@ -21,7 +21,7 @@ test:
 about:
   home: https://git-lfs.github.com/
   license: MIT
-  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
```

Comment:
These do appear in the wild like this though...

Reply:
Right. They are not an issue on input. When ruamel.yaml outputs this recipe it will remove the quotes since they are not needed. I change the input recipe but this is not needed. We use exact string matching on output so the change on the output version was needed for the tests.

Reply:
I added some quotes back to the input for a few recipes.